custom/icds_reports/templates/icds_reports/tableau.html

```diff
@@ -25,12 +69,68 @@
 <body>
     <nav class="navbar navbar-default">
         <div class="container-fluid">
-            <div class="nav navbar-nav">
+            <div class="navbar-header hq-header navbar-hq-main-menu">
+                <a href="{% if request|toggle_enabled:"USER_TESTING_SIMPLIFY" %}#{% else %}{% url "homepage" %}{% endif %}" class="navbar-brand">
+                    {% if CUSTOM_LOGO_URL %}
```

Comment:
In case reusing the code is not possible, can we get rid of the if-else and just use ICDS logo.